purchase_order_supplier_phone/models/purchase_order.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    supplier_phone = fields.Char(
+        related="partner_id.phone",
+        string="Supplier Phone",
```

Comment:
```suggestion
```
Didn't pre-commit tell you to remove this string for redundancy?

Reply:
pre-commit didn't tell to remove but I removed in the update commit